WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+Extensions.swift

```diff
@@ -246,7 +334,19 @@ private extension NotificationsViewController
         return WordPressAppDelegate.sharedInstance().simperium
     }
 
-    struct RatingSettings {
+    enum Properties {
+        static let sortKey          = "timestamp"
+    }
+
+    enum Filter: Int {
+        case None                   = 0
+        case Unread                 = 1
+        case Comment                = 2
+        case Follow                 = 3
+        case Like                   = 4
+    }
+
+    enum RatingSettings {
```

Comment:
@aerych this is a trick i've borrowed from Koke's book. I do agree with you, Structs *feel* way better. However, by using an Enum you prevent someone instantiating it: 

<img width="645" alt="screen shot 2016-07-07 at 13 52 35" src="https://cloud.githubusercontent.com/assets/1195260/16661612/1cce510c-444a-11e6-81b5-3361151ffd81.png">